tests/wrappers/vector/test_record_episode_statistics.py

```diff
@@ -56,16 +60,41 @@ def test_record_episode_statistics(num_envs, env_id="CartPole-v1", num_steps=100
         data_equivalence(wrapper_vector_reward, vector_wrapper_reward)
         data_equivalence(wrapper_vector_terminated, vector_wrapper_terminated)
         data_equivalence(wrapper_vector_truncated, vector_wrapper_truncated)
+        data_equivalence(wrapper_vector_info, vector_wrapper_info)
```

Comment:
I believe this isn't possible due to the episode time taken will vary between the two

This is the reason for the sequent code

A comment would be helpful

Reply:
Strangely, this test did not fail for me. I will move it back to where it was originally. Not sure why I moved it in the first place.